contracts/pm/TicketBroker.sol

```diff
@@ -11,8 +9,8 @@ import "./mixins/MixinWrappers.sol";
 contract TicketBroker is
     MixinContractRegistry,
     MixinReserve,
-    MixinTicketBrokerCore,
     MixinTicketProcessor,
+    MixinTicketBrokerCore,
```

Comment:
This is simply a more explicit way of writing `TicketBroker is MixinWrappers` since `MixinWrappers` already inherits all of the underlying contracts, whereby the order is `most "base-like" contract > most derived contract` or `low level > top level`

```solidity
     contract X {}
     contract A is X {}
     contract C is X, A {}
    ```